### PR TITLE
 chore: update flakebox  & nixpkgs

### PR DESCRIPTION
**Pull request by dpc**
"We didn't do it because it's easy. We did it because we thought it's easy."

### CODE REVIEW

File: deny.toml [hunk 1]
```diff
@@ -11,6 +11,9 @@
```
Comment (dpc) [inline deny.toml:11]: The format of this file changed, so I just regenerated it from a fresh template and ported existing directives until it works.

**Comment by dpc** [comment]
image


We'll have to live with this (or temporary disable), until https://github.com/DeterminateSystems/flake-checker-action/issues/47 is done.

**Review by bradleystachurski — DISMISSED** [review]

File: deny.toml [hunk 2]
```diff
@@ -96,8 +84,6 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-# The lint level for crates which do not have a detectable license
-unlicensed = "deny"
```
Comment (elsirion) [inline deny.toml:100]: Is that a new default or why don't we want it anymore?

Comment (elsirion) [reply deny.toml:100]: Apparently yes:

Screenshot 2024-11-26 at 12-14-11 config - cargo-deny

**Review by elsirion — DISMISSED** [review]

**Comment by elsirion** [comment]
There appears to be a problem with broken packages on Mac :(

**Comment by dpc** [comment]
> There appears to be a problem with broken packages on Mac :(

No one expected the MacOS to just work. :D

**Comment by dpc** [comment]
image

**Comment by dpc** [comment]
Possibly fixed by https://github.com/NixOS/nixpkgs/pull/358063 ?

File: .github/workflows/ci-nix.yml [hunk 3]
```diff
@@ -67,6 +67,13 @@ jobs:
       - name: Check `nix flake show`
         run: nix flake show .#
 
+      # Since we don't have capacity to build Darwin in PRs, at least make sure it evaluates
+      - name: Evaluate MacOS shell
+        run: nix eval --system aarch64-darwin .#devShells.aarch64-darwin.default
```
Comment (dpc) [inline .github/workflows/ci-nix.yml:72]: @elsirion @maan2003 TIL

**Comment by sarahec** [comment]
> Possibly fixed by [NixOS/nixpkgs#358063](https://github.com/NixOS/nixpkgs/pull/358063) ?

That's the plan.

**Review by bradleystachurski — DISMISSED** [review]

**Review by elsirion — DISMISSED** [review]

**Review by bradleystachurski — APPROVED** [review]

**Review by bradleystachurski — APPROVED** [review]

**Pull request merged**